screens/ConditionsResults.js

```diff
@@ -178,6 +219,23 @@ walleye:  [
           {expandedFish.includes(fishSpecies) && (
             <FlatList
               ListHeaderComponent={<View style={[flex_style.flex, flex_style.center, flex_style.column, margin_styles.vertical_space_md]}>
+                {reactIfView(currentTutorial == 'ConditionsGuide' && index == 0, <Tooltip
```

Comment:
I feel like these tutorial components could be put into one singular React component, with the props telling it which tutorial it is for. Putting repeated components like this into one would help for code repeatability & code reuse.

Reply:
I did this in the newest version, very good idea